packages/ui-popover/src/Popover/index.js

```diff
@@ -238,7 +239,48 @@ class Popover extends Component {
     /**
      * The content to be shown by the popover
      */
-    children: PropTypes.oneOfType([PropTypes.node, PropTypes.func])
+    children: PropTypes.oneOfType([PropTypes.node, PropTypes.func]),
+
+    /* eslint-disable react/require-default-props */
+    /**
+     * __Deprecated - use `isShowingContent` instead__
+     */
+    show: PropTypes.bool,
+    /**
+     * __Deprecated - use `defaultIsShowingContent` instead__
+     */
+    defaultShow: PropTypes.bool,
+    /**
+     * __Deprecated - use `color`__
+     */
+    variant: PropTypes.oneOf(['default', 'inverse']),
+    /**
+     * __Deprecated - use `shouldAlignArrow`__
+     */
+    alignArrow: PropTypes.bool,
+    /**
+     * __Deprecated - use `screenReaderLabel`__
+     */
+    label: PropTypes.string,
+    /**
+     * __Deprecated - use `shouldTrackPosition`__
+     */
+    trackPosition: PropTypes.bool,
+    /**
+     * __Deprecated - use `onShowContent`__
+     */
+    onShow: PropTypes.func,
+    /**
+     * __Deprecated - use `onHideContent`__
+     */
+    onDismiss: PropTypes.func,
+    /**
+     * __Deprecated - use `onShowContent` and `onHideContent`__
+     */
+    onToggle: PropTypes.func,
```

Comment:
Why did you add deprecated props back here?

Reply:
By mistake when rebasing next into my branch, should be fixed.